test/integration/generated/report/AutoRestReport.xcodeproj/xcshareddata/xcschemes/AutoRestReport-Package.xcscheme

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Scheme
+   LastUpgradeVersion = "9999"
+   version = "1.3">
+   <BuildAction
```

Comment:
I think this scheme is not needed. The default `swift package generate-xcodeproj` makes a bunch of schemes that we remove. I usually remove all the schemes that that command generates and just re-add the one that, in this case, would be `AutoRestReport`.

Reply:
Removed other schemes that we don't need.

Reply:
This one should be removed as well. There should only be an `AutoRestReport.xcscheme` which `generate-xcodeproj` doesn't create. That's why you remove all the ones it does create and then add this one. I can show you via screenshare if you want.